monai/transforms/compose.py

```diff
@@ -283,3 +283,36 @@ def inverse(self, data):
         _transform = self.transforms[index]
         # apply the inverse
         return _transform.inverse(data) if isinstance(_transform, InvertibleTransform) else data
+
+
+class RandomOrder(Compose):
```

Comment:
The compose's inverse is not compatible with this class, please double check

Reply:
Sorry, not sure what did you mean. Seems all the behaviors can be the same as before except for the order of the transforms. Thanks!

Reply:
for example with:
```py
fwd_data_1 = transform(data1)
fwd_data_2 = transform(data2)
fwd_inv = transform.inverse(fwd_data_1)
```

`transform(data2)` will shuffle `self.transforms`, and then calling `transform.inverse` it'll use the current `self.transforms` instead of inverting the original ones.. cc @Nic-Ma